packages/app/src/cli/services/release.ts

```diff
@@ -30,10 +31,19 @@ interface ReleaseOptions {
 export async function release(options: ReleaseOptions) {
   const {token, app, partnersApp} = await ensureReleaseContext(options)
 
-  const {versionsDiff, versionDetails} = await versionDiffByVersion(partnersApp.apiKey, options.version, token)
-
-  if (!options.force) await confirmReleasePrompt(partnersApp.title, versionsDiff)
-
+  const {extensionIdentifiersBreakdown, versionDetails} = await extensionsIdentifiersReleaseBreakdown(
+    token,
+    partnersApp.apiKey,
+    options.version,
+    app.specifications ?? [],
+  )
+  const confirmed = await deployOrReleaseConfirmationPrompt({
+    extensionIdentifiersBreakdown,
```

Comment:
why you don't use `configExtensionIdentifiersBreakdown` in this case? don't we want to prompt for that during release?